Sandbox/ManagerLayer.cpp

```diff
@@ -0,0 +1,70 @@
+#include "ManagerLayer.h"
+
+
+ManagerLayer::ManagerLayer(std::vector<Aquarius::Layer*> layers)
+	: Layer("Manager", true), m_ManagedLayers(layers)
+{}
+
+void ManagerLayer::onCreation()
+{
+	Aquarius::Application& app = *Aquarius::Application::get();
+
+	app.getEventHandler().subscribe(Aquarius::eventType::KeyPressedEvent,
+		[&](const Aquarius::Event& event)
+		{
+			onEvent(event);
+		});
+
+	for (const auto layer : m_ManagedLayers)
+	{
+		layer->deactivate();
+	}
+
+	if (!m_ManagedLayers.empty())
+	{
+		m_ManagedLayers.front()->activate();
+		m_ActiveLayerIndex = 0;
+	}
+}
+
+void ManagerLayer::onEvent(const Aquarius::Event& event)
```

Comment:
We need the esc to exit out of the application, with the mouse disabled it makes it difficult to navigate and close the window.

Reply:
@Burke-Daniel Esc key callback in window should handle this right?

Reply:
Yep, after reviewing this PR I went and did the pong one and made a note to put the esc back in the window haha